### PR TITLE
Make function legend in mprof plot unambiguous

### PR DESCRIPTION
**Pull request by maxhgerlach**
Previously, if the profiling was run on multiple same-named functions in
different classes or modules, the plot legend did not make clear which
bracket measured which function.

### CODE REVIEW

File: test/test_mprof.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+            "f.a.b": "f.a.b",
+            "g.a.b": "g.a.b",
+            "g.a.c": "a.c",
+            "b.c": "b.c",
+        }
+        result = mprof.function_labels(expected.keys())
+        self.assertEqual(expected,result)
+
+if __name__ == "__main__":
+    unittest.main()
```
Comment (Juanlu001) [inline test/test_mprof.py]: Newline missing

Comment (maxhgerlach) [reply test/test_mprof.py]: sure, thanks

**Comment by Juanlu001** [comment]
Thanks a lot1

**Comment by fabianp** [comment]
thanks @Juanlu001 for the review and merging!

**Pull request merged**